docs/sql-migration-guide.md

```diff
@@ -49,6 +49,8 @@ license: |
     * and the method `spark.catalog.refreshTable`
   In Spark 3.1 and earlier, table refreshing leaves dependents uncached.
 
+  - In Spark 3.2, the usage of `count(table.*)` is blocked to avoid producing ambiguous results. Because `count(*)` and `count(table.*)` will output differently if there is any null values. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.allowStarWithSingleTableIdentifierInCount` to `true`.
```

Comment:
`table` is a keyword, so `count(table.*)` is not parse-able under ansi mode. How about `count(tblName.*)`?